client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/EnrichDocumentationIT.java

```diff
@@ -77,4 +78,57 @@ public void onFailure(Exception e) {
         assertTrue(latch.await(30L, TimeUnit.SECONDS));
     }
 
+    public void testDeletePolicy() throws Exception {
+        RestHighLevelClient client = highLevelClient();
+
+        {
+            // Add a policy, so that it can be deleted:
+            PutPolicyRequest putPolicyRequest = new PutPolicyRequest(
+                "users-policy", "exact_match", List.of("users"),
+                "email", List.of("address", "zip", "city", "state"));
+            client.enrich().putPolicy(putPolicyRequest, RequestOptions.DEFAULT);
+        }
+
+        // tag::enrich-delete-policy-request
+        DeletePolicyRequest deletePolicyRequest =
+            new DeletePolicyRequest("users-policy");
+        // end::enrich-delete-policy-request
+
+        // tag::enrich-delete-policy-execute
+        AcknowledgedResponse deletePolicyResponse = client.enrich()
+            .deletePolicy(deletePolicyRequest, RequestOptions.DEFAULT);
+        // end::enrich-delete-policy-execute
+
+        // tag::enrich-delete-policy-response
+        boolean isAcknowledged =
+            deletePolicyResponse.isAcknowledged(); // <1>
+        // end::enrich-delete-policy-response
+
+        // tag::enrich-delete-policy-execute-listener
+        ActionListener<AcknowledgedResponse> listener = new ActionListener<>() {
+            @Override
+            public void onResponse(AcknowledgedResponse response) { // <1>
+                boolean isAcknowledged =
+                    deletePolicyResponse.isAcknowledged();
```

Comment:
this should be `response` not `deletePolicyResponse` methinks

Reply:
good catch. I copied this from the put policy test and there the same mistake was made too :)